### PR TITLE
Added Gutenberg extractor

### PR DESCRIPTION
**Pull request by J-Lemon**

### CODE REVIEW

File: MITADS/Gutenberg_Extractor.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import requests
+import urllib2
+import time
+from bs4 import BeautifulSoup
+
+urls = open( './books_list.txt', 'r' ).read().splitlines()
```
Comment (Mte90) [inline MITADS/Gutenberg_Extractor.py]: this file where is? can you add it to the pr?

**Comment by Mte90** [comment]
You didn't used https://github.com/c-w/gutenberg maybe because doing too many things?

File: MITADS/Gutenberg_Extractor.py (hunk 1 above)
Comment (J-Lemon) [reply MITADS/Gutenberg_Extractor.py]: I think that should be the final user to write the file.
Doing so the final user can chose wich book add and wich not.
I have included a small explaination how to write the file in the README.md .
I can include more details and a more explicative example how to compile books_list.txt in the readme.

**Comment by J-Lemon** [comment]
It's because that api just downloads the txt version of the book and not provide the functionalities to clean the text that we need.
In my implementation i take only the <p> tag from the html that in the most of case is the text we need.
For example all the chapters titles are automatically skipped because are contained in the <h2> tag.

File: MITADS/Gutenberg_Extractor.py (hunk 1 above)
Comment (Mte90) [reply MITADS/Gutenberg_Extractor.py]: This folder will include different exporter for different sources, so we need to specify better.
Also we need a default list, like in the pr :-)

**Pull request merged**